CalibTracker/SiPixelConnectivity/test/SiPixelFedCablingMapWriter_phase1.py

```diff
@@ -0,0 +1,56 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("MapWriter")
+process.load("CondCore.DBCommon.CondDBCommon_cfi")
+
+#process.load("Configuration.StandardSequences.GeometryDB_cff")
+process.load("Configuration.Geometry.GeometryExtended2017Reco_cff")
+
+
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
+# from Configuration.AlCa.GlobalTag import GlobalTag
+# works with condDB and condDBv2
+process.GlobalTag = GlobalTag(process.GlobalTag, '76X_upgrade2017_design_v8', '')
```

Comment:
@dkotlins we support a key in `autoCond.py` for the phase-I upgrades. You might want to use `phase1_2017_design` instead of an explicit GT